packages/web-components/src/tabs/tabs.ts

```diff
@@ -383,6 +383,21 @@ export class Tabs extends BaseTabs {
   @attr
   public size?: TabsSize;
 
+  /**
+   * Iterates over each tab element and sets its `disabled` attribute based on the `disabled` property of the `Tabs` component.
+   * If the `disabled` property is `true`, the `disabled` attribute of the tab element will be set to `'true'`; otherwise, it will be set to `'false'`.
+   */
+  protected setTabs(): void {
```

Comment:
Should this be done on `disabledChanged` given that's an attribute? I wouldn't enter the for loop unless disabled is true or disabled is changing to false.